shared_workflow/install_shared.py

```diff
@@ -160,7 +160,7 @@ def install_simulation(
         sim_params_dict[SimParams.stat_file.value] = stat_file_path
 
     nt = float(sim_duration) / root_params_dict['dt']
-    if not isclose(nt, int(nt)):
+    if not isclose(nt, round(nt)):
```

Comment:
Why round instead of int? Shouldn't nt always be an int?

Reply:
Because if float math approximates slightly below the correct value, then using int causes it to be rounded down to the next int, instead of to the nearest correct one